xenon-ftp-small/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM gliderlabs/alpine:3.1
```

Comment:
Why not from the `nlesc/xenon-alpine-base` image, then the xenon user and fixtures already exist?